klippy/extras/output_pin.py

```diff
@@ -45,12 +45,12 @@ def get_status(self, eventtime):
     def cmd_SET_PIN(self, gcmd):
         value = gcmd.get_float('VALUE', minval=0., maxval=self.scale)
         value /= self.scale
-        if value == self.last_value:
-            return
         print_time = self.printer.lookup_object('toolhead').get_last_move_time()
         print_time = max(print_time, self.last_value_time + PIN_MIN_TIME)
         if self.is_pwm:
-            self.mcu_pin.set_pwm(print_time, value)
+            cycle_time = gcmd.get_float('CYCLE_TIME',
+                self.mcu_pin._cycle_time, above=0., maxval=0.1)
```

Comment:
Out of curiosity, why is the cycle_time limited to a maximum of .1s?

Reply:
Not sure, for some reason I felt uncomfortable with "slow" PWM. Thinking about it now, and seeing that it is not limited in the config, I will remove the limit.